build.sbt

```diff
@@ -223,8 +223,34 @@ lazy val munitScalacheckJVM = munitScalacheck.jvm
 lazy val munitScalacheckJS = munitScalacheck.js
 lazy val munitScalacheckNative = munitScalacheck.native
 
+lazy val munitHedgehog = crossProject(JSPlatform, JVMPlatform, NativePlatform)
+  .in(file("munit-hedgehog"))
+  .dependsOn(munit)
+  .settings(
+    moduleName := "munit-hedgehog",
+    sharedSettings,
+    crossScalaVersions := List(scala213, scala212, scala211, dotty),
+    resolvers += "bintray-scala-hedgehog".at(
+      "https://dl.bintray.com/hedgehogqa/scala-hedgehog"
```

Comment:
It would be nice to have a hedgehog release on Maven Central.

Reply:
Agreed, there's an open issue for it and lead maintainer has been working it but not sure how close it is to being merged.